### ISSUE TITLE
Overriding defaults without helpers

### ISSUE THREAD

**Issue by devth** [mcasimir/meta-tags-helpers#3]
I see you can override defaults using these 4 helpers:

```
meta_title(value = nil)
meta_description(value = nil)
meta_image(value = nil)
meta_type(value = nil)
```

But how would one override `og:url` for instance? I'm assuming `<%= meta_tags %>` goes in my `application.html.erb` layout, and views would override any of the properties as needed.

**Comment by devth** [comment]
That would work. A helper that takes a full map and sets them all would be convenient.

In a view:

``` erb
<% set_meta "og:type" => "article", "og:url" => "http://foo" %>
```

**Comment by mcasimir** [comment]
Yes, better and simpler since a `deep_merge` would do all the work.. it doesn't seem a long task so i'm going to fix it now

**Comment by mcasimir** [comment]
It should be ok now, i'll release a new gem as soon as guys at `rubygems.org` will restore their service to a full functional status.

Thanks

**Comment by mcasimir** [comment]
Released now `gem "meta-tags-helpers", "~> 0.1.7"`.
